### PR TITLE
Main menu tweaks

### LINKED ISSUE #5: Create a basic main menu
Probably a temporary main menu, with no art/graphic design, just some buttons that lead to placeholder pages or are disabled?

- Play
- Achievements
- Settings
- About/credits?

Probably done by putting a bunch of https://api.flutter.dev/flutter/material/RaisedButton-class.html on main page, using https://api.flutter.dev/flutter/widgets/ListView-class.html for layout?

The old home screen should be where the new play button leads to.

### PR DESCRIPTION
**Pull request by JonMasson**
Clean up a few things from the main menu PR
- Refactor main menu buttons into separate widget
- Use relative imports where appropriate
- exclude VSCode settings from repo since we aren't all using it
- Remove unnecessary constructor from widget
- Add 'Credits' button since main menu looks better with 4 buttons :)

closes #5

### CODE REVIEW

File: lib/screens/main_menu/menu_button.dart [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+
+class MenuButton extends StatelessWidget {
+  final Function onPressed;
+  final String buttonText;
+
+  MenuButton(this.buttonText, {Function onPressed}):
+    this.onPressed = onPressed;
```
Comment (ghost) [inline lib/screens/main_menu/menu_button.dart]: Think you can one-line this like `MenuButton(this.buttonText, {this.onPressed} )` 😄

File: lib/screens/main_menu/menu_button.dart [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  MenuButton(this.buttonText, {Function onPressed}):
+    this.onPressed = onPressed; 
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(10.0),
+      child: RaisedButton(
+        onPressed: onPressed,
+        color: Colors.green,
```
Comment (ghost) [inline lib/screens/main_menu/menu_button.dart]: Could experiment with the theme a bit for this - from memory there are lots of things you can specify. For `Pocket Scat`, I think specifying `primaryColor` was enough to get my floating action button to automatically be purple.

**Review by ghost — APPROVED** [review]
LGTM

File: .gitignore [hunk 3]
```diff
@@ -18,7 +18,7 @@
 # The .vscode folder contains launch configuration and tasks you configure in
 # VS Code which you may wish to be included in version control, so this line
 # is commented out by default.
-#.vscode/
+.vscode/
```
Comment (ghost) [inline .gitignore:21]: Are there some files that we want to remove from source control along with this change?

Comment (JonMasson) [reply .gitignore:21]: Nope

**Review by kilikfan — APPROVED** [review]
Looks sensible.

**Pull request merged**